### ISSUE TITLE
What does valid ratio mean?

### ISSUE THREAD

**Issue by mkhoshle** [timmeinhardt/trackformer#63]
Hello,

In the Deformable Transformer, there is a variable called `valid_ratios` which is used based on the masks. `valid_ratios = torch.stack([self.get_valid_ratio(m) for m in masks], 1)`. If the masks are None in my case how am I supposed to calculate it?

Also, what is the purpose of `valid_ratios`? I could not find anything in the Trackformer and Original Deformable Detr paper.

I would appreciate it if you could clarify this.

**Comment by timmeinhardt** [comment]
These masks specify where padding was added to the image and are used to ignore those regions during the attention computation. If you do not have padding just initialize masks with zeros/False. See [here](https://github.com/timmeinhardt/trackformer/blob/d62d81023dbffb4a1820db39ce527b66df6d7b61/src/trackformer/util/misc.py#L309) for where the masks are generated.
